3.Tests/UI/StepDefinitions/Item/DeleteStepDefinition.cs

```diff
@@ -16,7 +16,7 @@ public class DeleteStepDefinitions : CommonSteps
 {
     private readonly HomePage _homePage;
     private readonly ScenarioContext _scenarioContext;
-    private string _expectedItemName = "";
+    private readonly string _expectedItemName = "";
```

Comment:
If this field is not being used should be removed please.

Reply:
Done